README.md

```diff
@@ -54,11 +54,20 @@ configuration.setParallelTesting(parallelTesting);
 configuration.setRunWithJenkins(runWithJenkins);
 configuration.setBuildNumber(buildNumber);
 // addidtional metadata presented on main page
-configuration.addClassifications("Platform", "Windows");
-configuration.addClassifications("Browser", "Firefox");
 configuration.addClassifications("Branch", "release/1.0");
+configuration.addClassifications("Browser", "Firefox");
+configuration.addClassifications("Platform", "Windows");
```

Comment:
any reason for this change?

Reply:
it aligns with screenshot I sent you...I can revert...the class I use to load properties (PropertiesConfiguration) maintains load order...doing it natively doesn't hence the reason to show this & plus the java comments below...

Reply:
revert as it seems to be not related to change you are working on